examples/test/BasicComponents_test.js

```diff
@@ -1,4 +1,4 @@
-// (C) 2007-2019 GoodData Corporation
+// (C) 2007-2020 GoodData Corporation
```

Comment:
we should have one simple case in `Basic Components`

Reply:
fiixed